recipes/diffpy-structure/run_test.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+
+import diffpy.Structure.tests
+assert diffpy.Structure.tests.test().wasSuccessful()
```

Comment:
Please add newline to the end of file.  It will make it binary-equal to run_test.py in the structure repository and
is also a correct, Unix-y way for ending text files.